src/components/chips/demoBasicUsage/index.html

```diff
@@ -5,7 +5,7 @@ <h2 class="md-title">Use a custom chip template.</h2>
 
     <form name="fruitForm">
       <md-chips ng-model="ctrl.roFruitNames" name="fruitName" readonly="ctrl.readonly"
-                md-removable="ctrl.removable" md-max-chips="5">
+                md-removable="ctrl.removable" md-max-chips="5" placeholder="Enter a fruit...">
         <md-chip-template>
           <strong>{{$chip}}</strong>
```

Comment:
In JAWS w/ IE11, upon adding a new chip, the screen reader announces "Left brace left brace dollar chip right brace right brace". Is the template being added to the dom before being linked?